src/web3/networkConfig.ts

```diff
@@ -71,7 +71,11 @@ interface NetworkConfig {
 }
 
 export type blockExplorerUrls = {
```

Comment:
could we make this Capital case for types? i think i commented on something similar before, sorry if i missed a reply there

Reply:
Sure, thanks! The other one was fixed, but this one I missed.